airframe-sql/src/main/scala/wvlet/airframe/sql/analyzer/TypeResolver.scala

```diff
@@ -365,6 +364,8 @@ object TypeResolver extends LogSupport {
           r.sourceColumn
         case a: Alias =>
           findSourceColumn(a.expr)
+        case s: SingleColumn =>
+          findSourceColumn(s.expr)
```

Comment:
`sourceColumns` had been lost when grouping key (and sort key) was specified by index.